doc/docs.md

```diff
@@ -377,6 +411,32 @@ if parser.token in [.plus, .minus, .div, .mult] {
 
 V optimizes such expressions, so both `if` statements above produce the same machine code, no arrays are created.
 
+Unlike Go language, semicolon `;` cannot be used for delimiting `if` conditions:
```

Comment:
With the same reason as "V" being better than "V language", "Go" would be better in this context.

Reply:
Updated,